mp/src/game/server/props.cpp

```diff
@@ -1611,6 +1618,7 @@ void CBreakableProp::Break( CBaseEntity *pBreaker, const CTakeDamageInfo &info )
 		{
 			bSmashed = true;
 		}
+#ifndef TF_DLL
 		else if ( pBreaker && dynamic_cast< CPropVehicleDriveable * >( pBreaker ) )
```

Comment:
`CPropVehicleDriveable` is not defined for this project, hence there is no type-info available to do the `dynamic_cast` with. I don't know what MSVC compiles this to, with gcc this results in a linker error due to missing symbol.
My best guess to implement this using `CBaseTFVehicle` is just below.